Polyfills/Console/Source/Console.cpp

```diff
@@ -36,18 +38,64 @@ namespace
 
     void InvokeCallback(Babylon::Polyfills::Console::CallbackT callback, const Napi::CallbackInfo& info, Babylon::Polyfills::Console::LogLevel logLevel)
     {
-        std::stringstream ss{};
-        for (size_t i = 0; i < info.Length(); i++)
-        {
-            if (i > 0)
-            {
-                ss << " ";
+        
+        std::string formattedString = "";
+        if (info.Length() > 0) {
+            formattedString = info[0].ToString().Utf8Value();
+            // check if this string has substitutions or not
+            std::regex toSub("(%[oOs])|(%(\\d*\\.\\d*)?[dif])");
```

Comment:
You don't want to create a new `regex` object each time. This can be const and put into an anonymous namespace in this file to make it only created once.